Teddykavooh/ReactApp
# Setting up my ReactApp

## What does this PR do?
```
Sets up a ReactApp from scratch.
```

## Description of Task to be completed?
```
Setting up a frontend environment using ReactJs.

```

## How should this be manually tested?

```
1. Clone the repo.
2. Run 'npm install' to install all dependancies.
3. Run 'npm start' and observe.

```

## Screenshots if available to support PR.

<img width="1280" alt="screen shot 2018-10-25 at 12 31 21" src="https://user-images.githubusercontent.com/39087913/47490685-ef16ce80-d851-11e8-9c41-f2f50f14d054.png">
